### ISSUE TITLE
NVDA doesn't read text borders in MS Word 2010

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2511]
**Reported by parham on 2012-07-07 18:31**
NVDA currently doesn't read the shapes in a Microsoft Word document. To reproduce this, do the following:
1. Go to MS Word.
2. Write "---" without quotes, or "***" (again without quotes).

The shape is there, but not announced.

**Comment by leonardder** [comment]
Here in word 2016, --- just produces --- and *** produces ***. Not sure what to do with this.

**Comment by Qchristensen** [comment]
If you type --- (three or more hyphens) then press ENTER, it creates a single line bottom border under the current paragraph.

If you type *** (three or more asterisks) then press ENTER, it creates a 3pt width dotted bottom border (visually a line of solid squares).

It's true that NVDA doesn't announce this, as it doesn't announce any borders, which is the subject of issue #2010

(Note: technically, I would expect that is a different issue to detecting and reporting "shapes", just to clear up any confusion from the initial comment above).

As a work around, press alt+h, b, o to open the borders and shading dialog.  The style, colour and width give information about the current border style, and if you tab past those three options, the state of the top, bottom, left and right check boxes indicate which of these borders are in use with the current style.

**Comment by ehollig** [comment]
Reporting of shapes is discussed in #5738 I believe. Closing as duplicate

**Comment by Qchristensen** [comment]
@ehollig - The name of this issue is confusing.  The issue is actually around reading borders, not shapes.  I've renamed the issue to clarify and reopened.

**Comment by Adriani90** [comment]
Closing as duplicate of #2010 which contains also a test document with different borders and shapes.
